src/app/features/tech-record/components/tech-record-history/tech-record-history.component.html

```diff
@@ -19,7 +19,9 @@
       <td class="govuk-table__cell">{{ techRecord.createdAt | date: 'dd/MM/yyyy HH:mm' }}</td>
       <td class="govuk-table__cell">
         <ng-container *ngIf="techRecord.createdAt !== currentRecord?.createdAt">
-          <a id="view-test-{{ vehicleTechRecord?.vin }}" href="/tech-records/{{ vehicleTechRecord?.vin }}/{{ convertToUnix(techRecord.createdAt) }}">View</a>
+          <a id="view-test-{{ vehicleTechRecord?.vin }}" href="/tech-records/{{ vehicleTechRecord?.vin }}/{{ convertToUnix(techRecord.createdAt) }}"
+            >View</a
+          >
```

Comment:
Having the `>`s on the next line looks really awkward 🙂 is this intended? Why not:
```suggestion
          <a id="view-test-{{ vehicleTechRecord?.vin }}" href="/tech-records/{{ vehicleTechRecord?.vin }}/{{ convertToUnix(techRecord.createdAt) }}">
            View
          </a>
```

Reply:
I think it's intentional by the formatter - some browsers will render the whitespace, but otherwise I agree that the closing `>` looks awkward  🙁